app/views/kits/index.html.erb

```diff
@@ -43,16 +43,16 @@
                   <%= label_tag "Also include inactive items" %>
                 </div>
               </div>
-              </div>
-              <div class="card-footer">
-                <%= filter_button %>
-                <%= cancel_button_to kits_path, { text: "Clear Filters" } %>
-                <span class="float-right">
-                  <%= new_button_to new_kit_path(organization_id: current_organization), { text: "New Kit" } %>
-                  </span>
-              </div>
-            <% end # form %>
-            </div>
+          </div>
+          <div class="card-footer">
+            <%= filter_button %>
+            <%= cancel_button_to kits_path, { text: "Clear Filters" } %>
```

Comment:
+:100: for this

Check up on line 37 (it was unchanged from the previous commit I think, and it looks like I overlooked it on #1976 :grimacing: ) The "Filters" wiki page has the docs on that interface, but you could probably copypasta it from a similar view.

Reply:
Thanks! Acutally on closer inspection, the filters don't really make sense here. Since all Kits have the same BaseItem it won't do anything. I am going to replace it with filter by name :) (which I think you have added in that PR)